komodo/__init__.py

```diff
@@ -35,14 +35,14 @@ def fixup_python_shebangs(prefix, release):
 
     bins_ = []
     # executables with wrong shebang
-    for bin_ in os.listdir(binpath):
-        try:
+    try:
+        for bin_ in os.listdir(binpath):
             with open(os.path.join(binpath, bin_), 'r') as f:
                 shebang = f.readline().strip()
             if _is_shebang(shebang):
                 bins_.append(bin_)
-        except Exception as err:
-            print('Exception in reading bin: %s' % err)
+    except Exception as err:
```

Comment:
`catch(Exception e) { e.printStackTrace(); }`

An exception here should fail the build imo.